app/services/rate_limit_checker.rb

```diff
@@ -97,6 +98,14 @@ def check_published_article_antispam_creation_limit
       SiteConfig.rate_limit_published_article_antispam_creation
   end
 
+  def check_published_comment_antispam_creation_limit
+    return true if user.created_at.before?(3.days.ago)
```

Comment:
I was a bit confused by how `published_article_antispam_creation` actually takes user age into account, as the description in `Config > Rate limits` suggests it should. Can anyone here enlighten me on that? cc @maestromac

Reply:
I'm also not quite sure what this is about either. Looks like @vaidehijoshi has briefly touched this area a bit, any idea?

Reply:
I tried to refresh my memory and looked back [at the original PR](https://github.com/forem/forem/pull/10686/files#diff-fccf03c2ae1e89b3cef4d8bd31718ab7d2b06ddabeafa900423ac0918dee46dcR34) where I added this, and voila I think I found it:
https://github.com/forem/forem/blob/c15120b902995c19bb7716a034802878434344e1/app/services/articles/creator.rb#L33-L41

Reply:
Thanks, @vaidehijoshi! I guess I'll follow a similar approach for comments then, better to be consistent.